bindings/python/test/test_bson.py

```diff
@@ -163,3 +178,32 @@ def test_simple(self):
         ]
         as_dict = {"data": [True, False, None, None, False, True]}
         self._run_test(docs, as_dict)
+
+
+class TestStringType(TestBsonToArrowConversionBase):
+    def setUp(self):
+        self.schema = Schema({"data": str})
+        self.context = PyMongoArrowContext.from_schema(self.schema)
+
+    def test_simple(self):
+        docs = [
+            {"data": "a"},
+            {"data": "dätá"},
+        ]
+        as_dict = {"data": ["a", "dätá"]}
+        self._run_test(docs, as_dict)
+
+
+class TestDecimal128StringType(TestBsonToArrowConversionBase):
```

Comment:
We also need to test the pandas/numpy conversion by testing Decimal128 with find_arrow/pandas/numpy

Reply:
Done